packages/icons-angular/src/lib/tabler-icon.module.ts

```diff
@@ -0,0 +1,15 @@
+import { ModuleWithProviders, NgModule } from '@angular/core';
+import { TablerIcons } from '../types';
+import { TABLER_ICONS, TablerIconProvider } from './tabler-icon.provider';
+
+@NgModule()
```

Comment:
Since standalone is the default. You might want to use the standalone. And not ngModule